src/openforms/prefill/contrib/family_members/haal_centraal.py

```diff
@@ -9,58 +9,44 @@
 
 from .constants import FamilyMembersTypeChoices
 from .filters import filter_members_by_age
-from .typing import FamilyMembersChildOptions, FamilyMembersPartnerOptions
-from .utils import convert_to_json_serializable
+from .typing import FamilyMemberOptions
 
 logger = structlog.stdlib.get_logger(__name__)
 
 
 def get_data_from_haal_centraal(
-    bsn: str,
-    options: FamilyMembersPartnerOptions | FamilyMembersChildOptions,
-) -> dict[str, list[NaturalPersonDetails]]:
+    bsn: str, options: FamilyMemberOptions
+) -> list[NaturalPersonDetails]:
```

Comment:
I had the dict in order to define specifically if it's `partners` or `children` for the frontend, but I guess we will know with this way as well.

Reply:
Discussed on Slack - this is still the same shape except the assignment is moved into the plugin code rather than the handler.